public/app/core/utils/connectWithReduxStore.tsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { store } from '../../store/store';
 
-export function connectWithStore(WrappedComponent, ...args) {
+export function connectWithStore(WrappedComponent: any, ...args: any[]) {
```

Comment:
Is there a better type for `WrappedComponent`? Any extenstion of `Component` or something similar?

Reply:
probably but not sure it's worth it